service/src/main/kotlin/app/cash/backfila/ui/pages/BackfillShowAction.kt

```diff
@@ -149,7 +149,11 @@ class BackfillShowAction @Inject constructor(
                   }
                   th(classes = "py-3 pl-8 pr-0 text-right font-semibold") {
                     scope = ThScope.col
-                    +"""Progress"""
+                    +"""# of ${"units (records, segments, bytes)"} Precomputed"""
```

Comment:
```suggestion
                    +"""# of ${record.unit.displayName} Precomputed"""
```

Why doesn't this actually use the relevant unit? Or just say `# Precomputed` and include the unit on the value in the cell below?

Reply:
Ah you are right, I deleted the actual units when testing. Modified!